MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -503,7 +543,8 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
 
     // Show the alert controller.
     MSLogDebug([MSDistribute logTag], @"Show update dialog.");
-    [alertController show];
+    [alertController replaceAlert:self.updateAlertController];
```

Comment:
Why do you need to keep this one? Alert is one time only so it can stay as a local variable.